pkg/sdk/testint/secrets_gen_integration_test.go

```diff
@@ -19,7 +19,7 @@ func TestInt_Secrets(t *testing.T) {
 
 	integrationId := testClientHelper().Ids.RandomAccountObjectIdentifier()
 
-	// "YYYY-MM-DD" or "YYYY-MM-DD HH-MI-SS" format has to be used, otherwise Snowflake returns error: "Invalid date/time format"
+	// "YYYY-MM-DD" or "YYYY-MM-DD HH:MI-SS" format has to be used, otherwise Snowflake returns error: "Invalid date/time format"
```

Comment:
Nit: just asking: shouldn't it be HH:MI:SS?

Reply:
It should, will fix this with next PR